src/index.html

```diff
@@ -22,6 +30,367 @@
+          />
+        </a>
+      </div>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a
+              class="header__link is-active"
+              href="#apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#samsung"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#smartphones"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li
+            data-qa="hover"
+            class="header__item"
+          >
+            <a
+              data-qa="nav-hover"
+              class="header__link"
+              href="#laptops-computers"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#tablets"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#photo"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="imac"
+        />
+        <div class="card__content">
+          <div class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__code">Product code: 195434</div>
+          <div class="card__feedback">
+            <div class="card__stars">
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star card__star--unfilled"></div>
+            </div>
+            <div class="card__review">Reviews: 5</div>
+          </div>
+          <div class="card__price">
+            <div class="card__price-name">Price:</div>
+            <div class="card__price-amount">$2,199</div>
+          </div>
+          <a
+            href="#"
+            data-qa="card-hover"
+            class="card__button"
+          >
+            BUY
+          </a>
```

Comment:
The 'BUY' button should be a `<button>` or `<input>` element. Screen readers will interpret it as a button and it will be included in the Tab order of the page. If you want to keep the `<a>` element, you should add `role='button'` to it.